docs/getting-started.md

```diff
@@ -187,10 +187,21 @@ CRDs, which provide advanced features including: policy priority, tiering, deny
 action, external entity, and policy statistics. For more information on usage of
 Antrea Network Policies, refer to the [Antrea Network Policy document](antrea-network-policy.md).
 
-### Traffic Encryption
+### Egress
 
-Antrea supports encrypting traffic between Linux Nodes using IPsec or WireGuard.
-To deploy Antrea with traffic encryption enabled, please refer to [this guide](traffic-encryption.md).
+Antrea supports specifying which egress (SNAT) IP the traffic from the selected
+Pods to the external network should use and which Node the traffic should leave
+the cluster from. For more information, refer to the [Egress document](egress.md).
+
+### Antrea Multi-cluster
```

Comment:
Do we list the features by alphabetic order? maybe the sequence of `Antrea Mult-cluster` and `Egress` need to be exchanged.

Reply:
How about let me move it to the bottom? I feel the feature is not stable enough yet to be listed as the first.